Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -856,13 +856,15 @@ def get_organizations_command(project_key=None, start="0", limit="50", account_i
     if account_id:
         body['accountId'] = account_id
 
-    if response := jira_req('GET', url, params=body, resp_type='json').get('values'):
-        [org.pop('_links') for org in response]
+    if response := jira_req('GET', url, params=body, resp_type='json'):
+        values = response.get('values')
+        [org.pop('_links') for org in values]
```

Comment:
This will mean we also don't include the `_links` values in `raw_response` outputs, since `values` is just a pointer for the same dict inside `response`.
What is this `_links` key for? I'm assuming they're pagination URLs?

If we do want to keep those for raw response outputs, we can do:
```suggestion
        raw_response = copy.deepcopy(response)
        values = response.get('values')
        [org.pop('_links') for org in values]
```

And then on the command results use `raw_response=raw_response`
(you'll need to import `copy` though, so you should make these changes on the IDE)